src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -2852,26 +2854,40 @@ namespace AirflowNetworkBalanceManager {
         CurrentModuleObject = "AirflowNetwork:MultiZone:Surface";
         for (int i = 1; i <= AirflowNetworkNumOfSurfaces; ++i) {
             if (MultizoneSurfaceData(i).SurfNum == 0) continue;
-            bool has_DOP_or_SOP{false};
+            bool has_Opening{false};
             // This is terrible, should not do it this way
             auto afe = solver.elements.find(MultizoneSurfaceData(i).OpeningName);
             if (afe != solver.elements.end()) {
                 auto type = afe->second->type();
-                has_DOP_or_SOP = (type == ComponentType::DOP) || (type == ComponentType::SOP);
+                has_Opening = (type == ComponentType::DOP) || (type == ComponentType::SOP) || (type == ComponentType::HOP);
             }
             // Obtain schedule number and check surface shape
-            if (has_DOP_or_SOP) {
+            if (has_Opening) {
                 if (Surface(MultizoneSurfaceData(i).SurfNum).Sides == 3) {
                     ShowWarningError(RoutineName + CurrentModuleObject + "=\"" + MultizoneSurfaceData(i).SurfName + "\".");
                     ShowContinueError(
                         "The opening is a Triangular subsurface. A rectangular subsurface will be used with equivalent width and height.");
                 }
+                // Venting controls are not allowed for an air boundary surface
+                if ((Surface(MultizoneSurfaceData(i).SurfNum).IsAirBoundarySurf) &&
+                    (MultizoneSurfaceData(i).VentSurfCtrNum != VentControlType::Const)) {
+                    ShowWarningError(RoutineName + CurrentModuleObject + "=\"" + MultizoneSurfaceData(i).SurfName + "\" is an air boundary surface.");
+                    ShowContinueError("Ventilation Control Mode = " + Alphas(4) + " is not valid. Resetting to Constant.");
+                    MultizoneSurfaceData(i).VentSurfCtrNum = VentControlType::Const;
+                    MultizoneSurfaceData(i).IndVentControl = true;
+                }
```

Comment:
It is good to reset the Const mode. It may be better to mention the reset in the IO Ref.

Reply:
Modified I/O Ref for Control Type = Constant:  If the surface is an air boundary (\hyperref[constructionairboundary]{Construction:AirBoundary}) then Constant is the only valid option (other options will throw a warning and be reset to Constant).